examples/septentrio_example.cc

```diff
@@ -1,28 +1,40 @@
 // Copyright (C) Point One Navigation - All Rights Reserved
 
-#include <iomanip>
 #include <iostream>
 
-#include "gflags/gflags.h"
-#include "glog/logging.h"
+#include <boost/asio.hpp>
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
 
 #include <point_one/polaris_asio_client.h>
+
 #include "septentrio_service.h"
 
 // Options for connecting to Polaris Server:
 DEFINE_string(
```

Comment:
We should eliminate these variables from the example as they don't expose both API and Polaris socket.

Reply:
Agreed.